WordPress/src/main/res/layout/stats_all_time_widget_configure_activity.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/constraintLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.coordinatorlayout.widget.CoordinatorLayout
+        android:id="@+id/coordinatorLayout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <fragment
+            android:id="@+id/fragment_container"
```

Comment:
This is pretty minor, but maybe we could stick to a single naming style within layout file (camelCase?)

Reply:
I haven't noticed the camel case :-), this class is a copy&paster and I was just adjusting the content and the toolbar. I think IDs are usually snake case, right? I'm ok with both 👍 . I've changed the camel case to snake case in the parent layout